forge/routes/api/project.js

```diff
@@ -128,6 +128,7 @@ module.exports = async function (app) {
                     applicationId: { type: 'string' },
                     projectType: { type: 'string' },
                     stack: { type: 'string' },
+                    flowTemplate: { type: 'string' },
```

Comment:
The frontend currently calls this `flowTemplateId` since it's an ID for a template rather than a flowTemplate itself

Reply:
Yeah, I paused on this. The existing properties `projectType`, `stack` and `template` are all also ids but not labelled as such. I erred on local consistency... but can be swayed to introduce `flowTemplateId`.

Reply:
I’d like our convention going forward to be use the word ID if it’s an ID. It’s much more explicit about what is expected.

Reply:
I'm okay with that. Will updated